vuepress/docs/next/tutorials/create/ms/update-data-model.md

```diff
@@ -43,25 +45,26 @@ relationship OneToMany {
    Conference to Session
 }
 ```
-In this case we've added two fields to the Conference entity, introduced the Session entity plus an enum, and added a mapping between the two entities. This is the view you'll get in JDL-Studio for the updated data model.
+The above adds two fields to the Conference entity, introduces the Session entity plus an enum, and creates a mapping between the two entities. Below is the updated data model in JDL-Studio:
 
 ![conference.jdl](./img/jhipster-jdl.png)
 
-4. Now import the jdl file into your application. You may be asked if you want to generate MFEs depended on your options when first generating the project.
+4. Import the JDL file into your application. The options selected during project generation determine if you are prompted with the option to generate MFEs.
 ```
 ent jhipster import-jdl conference.jdl
 ```
-If you kept the original project structure, this step will update your data model, add entries to Liquibase so the database schema can be upgraded at deploy time, add new service methods to your microservice, add fields to your MFEs, etc.
+If the original project structure has been retained, this step will update your data model, add entries to Liquibase to upgrade database schema during deployment, add service methods to your microservice, add fields to your MFEs, etc.
```

Comment:
the original opening phrase of the sentence reads more direct and action oriented >> "If you kept the original project structure..."

Reply:
so, i didn't like this because you can also be using an existing project, in which case *you* may not have touched anything. how do you even know if you're working with the original structure? i thought it was best to cover all bases